crates/wasm/Cargo.toml

```diff
@@ -20,11 +20,15 @@ ambient_gpu = { path = "../gpu" }
 ambient_renderer = { path = "../renderer" }
 ambient_procedurals = { path = "../procedurals" }
 
+ambient_guest_bridge = { path = "../../shared_crates/guest_bridge", default-features = false, version = "0.3.0-dev" }
+
 ambient_project = { path = "../../shared_crates/project" }
 ambient_shared_types = { path = "../../shared_crates/shared_types", features = [
     "native",
 ] }
 
+webbrowser = "0.8.10"
```

Comment:
Might be a good idea to use `open`, which we have a workspace dep for. Not sure if it'd be a security risk, though - does `open::that("explorer.exe")` work?

Reply:
> Might be a good idea to use `open`, which we have a workspace dep for. Not sure if it'd be a security risk, though - does `open::that("explorer.exe")` work?

`open` works.
I am not on Windows so I am not sure of the latter. Isn't `that` only used for URL? https://docs.rs/open/latest/open/